var/spack/repos/builtin/packages/py-easybuild-framework/package.py

```diff
@@ -11,12 +11,17 @@ class PyEasybuildFramework(PythonPackage):
     for (scientific) software on HPC systems.
     """
 
-    homepage = 'http://hpcugent.github.io/easybuild/'
-    url      = 'https://pypi.io/packages/source/e/easybuild-framework/easybuild-framework-3.1.2.tar.gz'
+    homepage = 'https://easybuilders.github.io/easybuild'
+    url      = 'https://pypi.io/packages/source/e/easybuild-framework/easybuild-framework-4.0.0.tar.gz'
+    maintainers = ['boegel']
 
+    version('4.0.0', sha256='f5c40345cc8b9b5750f53263ade6c9c3a8cd3dfab488d58f76ac61a8ca7c5a77')
     version('3.1.2', '283bc5f6bdcb90016b32986d52fd04a8')
 
-    depends_on('python@2.6:2.8', type='run')
-    depends_on('py-setuptools', type=('build', 'run'))
-    depends_on('py-vsc-base@2.5.4:', when='@2.9:', type='run')
-    depends_on('py-vsc-install', type='run')  # only required for tests (python -O -m test.framework.suite)
+    depends_on('python@2.6:2.8', when='@:3', type=('build', 'run'))
+    depends_on('python@2.6:2.8,3.5:', when='@4:', type=('build', 'run'))
+    depends_on('py-setuptools', when='@:3', type=('build', 'run'))
+    depends_on('py-vsc-base@2.5.4:', when='@2.9:3', type='run')
```

Comment:
Does the `:3` mean "any 3.x version" or "any 2.x version, not including 3"?

EasyBuild v3.x required `setuptools`/`vsc-base`/`vsc-install` (so does v2.x), EasyBuild v4.0.0 no longer requires `setuptools`/`vsc-base`/`vsc-install`

Reply:
It means up to and including any v3.x release.

Reply:
OK, then it's correct, wasn't sure about the semantics.